Sync/DAO/Value/ReferenceValueDAO.php

```diff
@@ -18,20 +18,40 @@ class ReferenceValueDAO
     private $value;
 
     /**
-     * @return int
+     * @var string
      */
-    public function getValue()
+    private $type;
+
+    /**
+     * @return int|null
+     */
+    public function getValue(): ?int
     {
         return $this->value;
     }
 
     /**
      * @param int
      */
-    public function setValue($value)
+    public function setValue(int $value)
```

Comment:
```suggestion
    public function setValue(int $value): void
```
The other setter returns void as well